sql/migrations/pg_notifier_queue_always/000001_alter_payload_jsonb_to_text.up.tmpl

```diff
@@ -0,0 +1,18 @@
+{{ if .config.GetBoolVar false "PgNotifier.enableJsonbToText" }}
+    DO $$
+    BEGIN
+      IF EXISTS (
+        SELECT 1 FROM information_schema.columns
+        WHERE table_name = 'pg_notifier_queue'
+          AND column_name = 'payload'
+          AND data_type != 'text'
+      ) THEN
+        RAISE NOTICE 'Altering column "payload" in table "pg_notifier_queue" from % to text',
+          (SELECT data_type FROM information_schema.columns WHERE table_name = 'pg_notifier_queue' AND column_name = 'payload');
```

Comment:
Instead of querying `information_schema` twice, how about we query it once and based on the output we write an if-else block ?